.prettierrc.yml

```diff
@@ -0,0 +1,4 @@
+trailingComma: "es5"
+semi: false 
+singleQuote: true
+tabWith: 2
```

Comment:
Willing to add `proseWrap: always`, by any chance? And maybe bump `printWidth` up to 100 or 120?

Reply:
for `printWidth` my only problem is that I constantly split my editors vertically in `vscode`.  I also often put a terminal and vscode in the same monitor splitted vertically across the window 

- 120 is definitely too long and will give me a lot of hassle.
- 100 is a bit better, but still gives me some hassle. 

See screenshot:
![image](https://user-images.githubusercontent.com/1408006/116808144-3c501700-ab05-11eb-9d06-3fef9ebdfd5b.png)

Reply:
Fair enough — keeping `printWidth` at 80 is fine. Any chance for `proseWrap`?

Reply:
Ah sorry, I see now that you added it. Was using the wrong view.